operatorcourier/cli.py

```diff
@@ -2,11 +2,17 @@
 import sys
 from operatorcourier import api
 
+sys.tracebacklimit = 0  # this suppresses stacktrace of the CLI tool in the command line
+
+
 def main():
     """Generate the CLI bits
     """
-    parser = _CliParser()
-    parser.parse()
+    try:
+        parser = _CliParser()
+        parser.parse()
+    except Exception:  # Exception raised to CLI should not be thrown to users,
+        exit(1)        # it should just be captured by logs
```

Comment:
I still don't understand why we need to use the `exit()` syscall here. Can't we just print a message and return after printing?

Reply:
What I'm thinking is that
  - we should not make exit call within modules, which we have discussed previously
  - the reason why I didn't just print a message and return is that, by doing this the exit status will be 0. But I want the user to know that the CLI program did not exit correctly.

Any thoughts on that?

Reply:
Right, this is the cli. This looks fine.